docs/src/txFlagIndices.md.ejs

```diff
@@ -0,0 +1,15 @@
+## accountSetFlags
```

Comment:
Is it possible to change the name of this file to accountSetFlagIndices.md.ejs instead of txFlagIndices.md.ejs? (Or would that mess up the docgen?)